data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -173,23 +133,21 @@ public Object process(org.apache.hadoop.hive.ql.lib.Node nd, Stack<org.apache.ha
 		ASTNode pt = (ASTNode) nd;
 		switch (pt.getToken().getType()) {
 			case HiveParser.TOK_QUERY:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 44 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1151)